src/migrations/1647573802137_mempool_digest.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate';
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+async function pgServerMajorVersion(pgm: MigrationBuilder): Promise<number | undefined> {
+  const result = await pgm.db.query(`SHOW server_version`);
```

Comment:
Just did some searching around this command.. WTF? Strange that this is essentially a free form field as opposed to a strict semver or similar.

Would it be better to check for a specific features availability? Could help in the future if we have features dependent on enabled modules as opposed to server version. Something like 

```ts
async function isBitXorAvailable(pgm: MigrationBuilder) {
  try {
    await pgm.db.query('select bit_xorr(1)');
    return true;
  } catch (error: any) {
    if (error.code === '42883' /* UNDEFINED_FUNCTION */) {
      return false;
    }
    throw error;
  }
}
```

Reply:
Great idea. I'll do this